site/docs/operations/docker.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Hazelcast Jet with Docker
+description: How to use Jet in a Docker environment
+---
+
+This section provides comprehensive coverage of usage of Hazelcast Jet
+inside a Docker environment. While it gives an explanation of all
+options and parameters used in the examples it is recommended to be
+familiar with Docker, visit e.g.  the [official
```

Comment:
.... used in the examples, it is recommended ...

... with Docker; you can visit the [official ...